core/config/locales/en.yml

```diff
@@ -134,6 +134,10 @@ en:
         name: Name
         preference_source: Preference Source
         type: Provider
+      spree/price:
+        currency: Currency
+        amount: Price
+        is_default: Currently Valid
```

Comment:
I would love to see time-based pricing. That aside, `is_default` is the most poorly named boolean I've seen so far: It's `true` be default, and adding a new price for a given currency will set all other prices for that currency/variant to `is_default: false`. So "Currently valid" is actually a pretty accurate translation for what it does.